lib/twingly/url.rb

```diff
@@ -6,7 +6,19 @@
 require_relative "url/error"
 require_relative "version"
 
-PublicSuffix::List.private_domains = false
+PublicSuffix::List.default = PublicSuffix::List.parse(
+  File.read(PublicSuffix::List::DEFAULT_LIST_PATH), private_domains: false)
+
+# Add ASCII form of all internationalized domain names to the public suffix list
+PublicSuffix::List.default.
+  map { |rule| Addressable::IDNA.to_ascii(rule.value) }.
```

Comment:
We could try with both "pure" and "libidn" implementations just so we know we won't run into something exciting once we drop libidn.